docs/content/envoy/workflows.en.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Envoy Workflows
+date: 2024-07-31T13:31:11-05:00
+lastmod: 2024-07-31T13:31:11-05:00
+description: "How Travel Rule information exchanges work in the Envoy UI and API"
+weight: 20
+---
+
+A brief summary of a travel rule compliance exchange with Envoy is as follows:
+
+1. The originating VASP identifies a counterparty either via a Travel Address (TRP) or by using the counterparties list (TRISA).
+2. Optionally, an address confirmation message is sent to the counterparty to confirm the counterparty controls the specified wallet address.
+3. The originating VASP creates a transaction then sends a secure envelope to the counterparty via TRISA or TRP (or email) so that they can review it.
+4. The counterparty VASP reviews the information then accepts it if it meets their compliance requirements or rejects it if it doesn't.
+5. If accepted, the originating VASP completes the on-chain transaction, then sends a completion message back to the counterparty with the transaction ID.
+
+There are four basic workflows:
+
+- Accepted and Completed
+- Repairs Requested
+- Rejected and/or Canceled
+
+## Accepted and Completed
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant Oc as Originator User
+    participant O as Originator Envoy
```

Comment:
Would adding `Envelope` after or below `Envoy` (if possible) help add a distinction between the `transaction` and `envelope` states? That may be redundant though since `RPC` is included in the diagrams when above the envelope states.

Reply:
That's a good idea, I'll do that.